### PR TITLE
Add minor features

### PR DESCRIPTION
**Pull request by energyLS**
## Changes proposed in this Pull Request

- Fixes the scaling bug in shipping
- Introduces the export wildcard

## Checklist

- [ ] I tested my contribution locally and it seems to work fine.
- [ ] Code and workflow changes are sufficiently documented.
- [ ] Newly introduced dependencies are added to `envs/environment.yaml` and `envs/environment.docs.yaml`.
- [ ] Changes in configuration options are added in all of `config.default.yaml`, `config.tutorial.yaml`, and `test/config.test1.yaml`.
- [ ] Changes in configuration options are also documented in `doc/configtables/*.csv` and line references are adjusted in `doc/configuration.rst` and `doc/tutorial.rst`.
- [ ] A note for the release notes `doc/release_notes.rst` is amended in the format of previous release notes, including reference to the requested PR.

### CODE REVIEW

File: scripts/override_respot.py [hunk 1]
```diff
@@ -95,6 +102,21 @@ def override_values(tech, year, dr):
     else:
         print("No RES potential techs to override...")
 
+    if snakemake.config["custom_data"]["add_existing"] and snakemake.wildcards["planning_horizons"] == '2030':
+            
+            n.generators.loc["MAR010003 onwind", "p_nom_min"] = 50
```
Comment (energyLS) [inline scripts/override_respot.py:107]: The data listed here should be moved to config or input data file

File: scripts/prepare_sector_network.py [hunk 2]
```diff
@@ -964,8 +993,8 @@ def add_shipping(n, costs):
         co2 = (
             shipping_oil_share
             * ports["p_set"].sum()
-            * 1e6
-            / 8760  # *n.snapshot_weightings.objective[0] #TODO change the way pset is sampled here
+    
+            #/ 8760  # *n.snapshot_weightings.objective[0] #TODO change the way pset is sampled here
```
Comment (energyLS) [inline scripts/prepare_sector_network.py:997]: we need to fix this, either divide by 8760 or `n.snapshot_weightings.objective[0]`, but doing neither leads to a bug

File: scripts/prepare_sector_network.py [hunk 3]
```diff
@@ -2046,7 +2123,6 @@ def add_custom_water_cost(n):
             index_col=0,
         )
         water_costs = water_costs.filter(like=country, axis=0).loc[nodes]
-
         electrolysis_links = n.links.filter(like=country, axis=0).filter(
             like="lectrolysis", axis=0
```
Comment (energyLS) [inline scripts/prepare_sector_network.py:2127]: The term "lectrolysis" seems to be a typo?

File: scripts/solve_network.py [hunk 4]
```diff
@@ -272,7 +289,7 @@ def add_co2_sequestration_limit(n, sns):
 
     lhs = linexpr((1, vars_final_co2_stored)).sum()
     rhs = (
-        n.config["sector"].get("co2_sequestration_potential", 200) * 1e6
+        n.config["sector"].get("co2_sequestration_potential", 5) * 1e6
     )  # TODO change 200 limit (Europe)
```
Comment (energyLS) [inline scripts/solve_network.py:293]: Changed to `5`, so now we can delete this "Todo"?

**Comment by energyLS** [comment]
@Hazem-IEG I have opened up a PR with your changes and reviewed them. I have some minor comments but nothing critical I guess. 
One important thing though: I think you have incorporated many changes in `config.yaml` (e.g. h2export, etc,...) but this file is in gitignore. Could you copy the changes you have made in `config.yaml` to `config.default.yaml` and push `config.default.yaml`? Then I can proceed in getting the CI running. Thanks :)

**Comment by energyLS** [comment]
@Hazem-IEG I have made a couple of updates in the `config.pypsa-earth.yaml` and `config.test1.yaml` and tested it. Now it looks all good and runs on my local machine, I expect the CI to run as well. Once it does, could you please add an "Approving review"? -> Just approve it, could be done in seconds. Since I opened up the PR, my review doesn't count and the merge is blocked.

**Comment by energyLS** [comment]
@Hazem-IEG I have fixed it :) Now it is all ready to merge, I just need your approving review. Thanks!

**Review by Hazem-IEG — APPROVED** [review]

**Pull request merged**